sotodlib/site_pipeline/make_hwp_solutions.py

```diff
@@ -62,21 +63,22 @@ def get_parser(parser=None):
     return parser
 
 def main(
-    context: str, 
-    HWPconfig: str, 
-    output_dir:Optional[str] = None, 
-    verbose:Optional[int] = 2,
-    overwrite:Optional[bool] = False,
-    query:Optional[str] = None,
-    min_ctime:Optional[float] = None,
-    max_ctime:Optional[float] = None,
-    obs_id:Optional[str] = None,    
-    logger=None,
+    context: str,
+    HWPconfig: str,
+    output_dir: Optional[str] = None,
+    verbose: Optional[int] = 2,
+    overwrite: Optional[bool] = False,
+    query: Optional[str] = None,
+    min_ctime: Optional[float] = None,
+    max_ctime: Optional[float] = None,
+    obs_id: Optional[str] = None,
+    logger = None,
+    load_h5: Optional[bool] = False,
```

Comment:
logger needs to be the last argument passed to main (it's a weird prefect thing)